sessions/js-unit-testing/get-animal/index.test.js

```diff
@@ -0,0 +1,14 @@
+import {getAnimal} from './index.js';
+
+test('getAnimals() returns a string "I really like cats!" if called with the argument "cat".', () => {
+	const result = getAnimal('cat');
+	expect(result).toBe('I really like cats!');
```

Comment:
The difference between "I really like cats!" and "I like dogs!" is quite subtile.

It might be a good idea to change the sentence so it is more clear to students they need to handle the input `"cat"` different than other inputs. Idea:

> "I totally love cats!"